Makefile

```diff
@@ -0,0 +1,8 @@
+format:
+	isort --fss -w 88 .
+	black .
+
+lint:
+	isort --fss -w 88 --check-only .
+	black . --check
+	flake8 kuwala/core kuwala/common
```

Comment:
will sort imports, format codebase and use flake8 to do some simple check

Reply:
only check `kuwala/core and common` because there's lot's of exception in `pipelines`
want to punt on this in this PR 😛